packages/charts/examples/components/demo/chart-types/area/chart-docs-area.module.ts

```diff
@@ -18,6 +20,7 @@ import { AreaChartVariableBaselineExampleComponent } from "./area-chart-variable
 import { AreaChartVerticalExampleComponent } from "./area-chart-vertical/area-chart-vertical-example.component";
 import { ChartDocsAreaComponent } from "./chart-docs-area.component";
 import { AreaChartTestComponent } from "./test/area-chart-test.component";
+/* eslint-enable max-len */
```

Comment:
Isn't this on globally? why is it needed here?

Reply:
I actually disable it at the top of the file. This is just re-enabling it afterwards.